### PR TITLE
Optimize docker image

### PR DESCRIPTION
**Pull request by RoyTakanen**
**Disclaimer:** I have not tested this code (as long as there is no merge conflicts everything should work :P)

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 while [ 1 ];
 do
-    /usr/local/cargo/bin/diesel database setup && break;
+    /app/iesel database setup && break;
```
Comment (lajp) [inline entrypoint.sh:4]: this seems like a type

**Review by lajp — CHANGES_REQUESTED** [review]

File: entrypoint.sh (hunk 1 above)
Comment (lajp) [reply entrypoint.sh:4]: *typo

Comment (RoyTakanen) [reply entrypoint.sh]: fixed

**Comment by lajp** [comment]
LGTM! Thank you for this contribution!

**Comment by lajp** [comment]
Merging now

**Review by lajp — APPROVED** [review]

**Pull request merged**